aspnetcore/data/ef-rp/migrations.md

```diff
@@ -297,7 +297,7 @@ EF Core uses the `__MigrationsHistory` table to see if any migrations need to ru
 ## Troubleshooting
 
 Download the [completed app](
-https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu21snapshots/part-4-migrations).
+https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/data/ef-rp/intro/samples/cu21snapshots/Part-4-Migrations).
```

Comment:
I'd revert this and the other case changes. Only misconfigured servers have case sensitive URL.

Reply:
I checked on the "SameSite" one ... *Indeed!* They ain't doing the right thing on that server. It 404's if lowercase.